compiler/compile/OMRCompilation.cpp

```diff
@@ -787,11 +787,16 @@ OMR::Compilation::getOSRInductionOffset(TR::Node *node)
    if (self()->getOSRTransitionTarget() != TR::postExecutionOSR)
       return 0;
 
+   if (node->getOpCodeValue() == TR::treetop || node->getOpCode().isCheck())
```

Comment:
Throughout OSR functions an assumption is made that a treetop or check will have a call as a child, as this is the only treetop/check that can pass isPotentialOSRPoint. As this was the same assumption that resulted in the defaults this commit is attempting to address, I will add the check for calls as well.